Package.swift

```diff
@@ -75,6 +74,19 @@ let package = Package(
                 .product(name: "SwiftyJSON", package: "SwiftyJSON"),
             ]
         ),
+        .target(
+            name: "CacheStorage",
+            dependencies: [
+                "ScipioKitCore"
+            ]
+        ),
+        .target(
+            name: "ScipioKitCore",
```

Comment:
Implementations look good.
But I feel `ScipioKitCore` is a bit of an ambiguous name. Furthermore, it's problematic that CacheStorage depends on this. I'll think of a better name.